packages/lib/modules/transactions/transaction-steps/useSignPermit2Step.tsx

```diff
@@ -62,25 +64,25 @@ export function useSignPermit2Step(params: AddLiquidityPermit2Params): Transacti
   function SignPermitButton() {
     return (
       <VStack width="full">
-        {error && <BalAlert content={error} status="error" />}
+        {error ? <BalAlert content={error} status="error" /> : null}
         {!isConnected && <ConnectWallet isLoading={isLoading} width="full" />}
-        {shouldChangeNetwork && isConnected && (
+        {shouldChangeNetwork && isConnected ? (
           <NetworkSwitchButton {...networkSwitchButtonProps} />
-        )}
-        {!shouldChangeNetwork && isConnected && (
+        ) : null}
```

Comment:
Why ? then null instead of just &&?

Reply:
Fixed. I did this during the monorepo + eslint migration and somehow my lint auto formatted did this by mistake.